index.html

```diff
@@ -914,14 +1036,66 @@ <h4>serializeDerivedProofValue</h4>
 Section <a href="#compresslabelmap"></a>, passing `labelMap` as the parameter.
             </li>
             <li>
-Initialize a byte array, `proofValue`, that starts with the BBS disclosure
-proof header bytes `0xd9`, `0x5d`, and `0x03`.
-            </li>
-            <li>
+ Depending on the value of |featureOption| do the following:
+              <ol class="algorithm">
+                <li>
+If |featureOption| equals `"baseline"`:
+                  <ol class="algorithm">
+                    <li>
+Initialize |proofValue| to start with the
+disclosure proof header bytes `0xd9`, `0x5d`, and `0x03`.
+                    </li>
+                    <li>
+Initialize |components| to an array with elements containing the values of
+|bbsProof|, |compressedLabelMap|, |mandatoryIndexes|, |selectiveIndexes|, and
+|presentationHeader|.
+                    </li>
+                  </ol>
+                </li>
+                <li>
+If |featureOption| equals `"anonymous_holder_binding"`:
```

Comment:
Using snake_case here is inconsistent with our other names, so I'd prefer it be camel case or hyphenated (or whatever else matches our current style elsewhere), but again wouldn't hold up the PR for that.

Reply:
Hi @dlongley, this is a bit of a mess and extends to variable names, and the IETF specs too (mixing camel and snake case). I would be nice to choose *camel* or *snake* case (or whatever) throughout or have a established naming convention for functions, variable, constants,  constant strings, etc. I'm more than happy to clean up everything according to a nice convention. Does W3C have anything written? I'm not strongly optioned on what the conventions should be.